[7.x][DOCS] Adds ML related items to release highlights (#64299)
elastic/kibana: docs/release-notes/highlights-7.7.0.asciidoc

[role="screenshot"]
image:release-notes/images/7-7-maps-distance-filter.png[Distance filters in Elastic Maps]

[float]
[[file-uploader-highlights]]

> === File uploader now recommends Filebeat config

Reviewer: AFAIK this is still referred to as the Data Visualizer
```suggestion
=== {data-viz} now recommends {filebeat} config
```

> experimental[] The file upload UI now has the ability to recommend a Filebeat 

Reviewer: ```suggestion
experimental[] If you upload a file to the *{data-viz}*, it now has the ability to recommend a {filebeat} 
```

> experimental[] If you upload a file to the *{data-viz}*, it now has the ability to recommend a {filebeat} 

Reviewer 2: Could you tighten up the first sentence a bit? I'm getting stuck on the `it` part. Maybe something like:

When you upload a file to *{data-viz}*, a {filebeat} configuration is recommended for you to use.

> configuration based on the uploaded file. It can be used to add more data of 

Reviewer: ```suggestion
configuration. You can then use {filebeat} to add more data of 
```

the same structure to the same index that was created during an initial upload.

> [role="screenshot"]

Reviewer 2: Since the side navigation is constantly changing, we don't include it in our screen captures.

image:release-notes/images/7-7-file-uploader-filebeat.png[Recommended Filebeat config in File uploader]

> 

Reviewer: I think it would be helpful to add a link too:
```suggestion
For more information about *{data-viz}*, see <<xpack-ml>>.
```

> For more information about *{data-viz}*, see <<xpack-ml>>.

Reviewer 2: To meet accessibility requirements, we are no longer using `see` in our documentation. Instead, we are using `refer to`. For example:

For more information about *{data-viz}*, refer to <<xpack-ml>>.

Reviewer: Oops, I hadn't heard of that change and reverted it when I fixed a broken link in the backport PR.
